xlators/cluster/dht/src/dht-common.c

```diff
@@ -2003,6 +2003,7 @@ dht_lookup_linkfile_create_cbk(call_frame_t *frame, void *cooie, xlator_t *this,
     cached_subvol = local->cached_subvol;
     conf = this->private;
 
+    DHT_CHECK_AND_UNREF_TMP_FD(local);
```

Comment:
I think we can just leave the fd on local->fd and later the dht_local_wipe should take care of unreffing it. I can't think of any issues keeping the fd in local. Did you see any issue?

Reply:
> I think we can just leave the fd on local->fd and later the dht_local_wipe should take care of unreffing it. I can't think of any issues keeping the fd in local. Did you see any issue?

clearing local->fd in dht_local_wipe is more reasonable, I need some time to investigate the cause of EBADFD, maybe it's related to my testing methods.